src/lib/SparkDB.java

```diff
@@ -58,7 +54,7 @@ public void create(ArrayList<String> headers) throws Exception {
      * @param Crypt_Key The decryption key
      */
     public void readFromFile(String filename, String Crypt_Key) throws Exception {
-        BufferedInputStream BIF = new BufferedInputStream(new FileInputStream(new File(filename)), 4096);
+        BufferedInputStream BIF = new BufferedInputStream(new FileInputStream(filename), 4096);
```

Comment:
Please double check if it works the same without File class initialization

Reply:
Yes, it does